ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -437,6 +437,77 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 0 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: ocp-4.5
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-release-master-ocp-4.5-e2e-vsphere
```

Comment:
i think you should only be adding the 4.6 variant to "master"

Reply:
(I guess the generator did this?)

Reply:
yeh this was generated by `make jobs`.

Then I manually created the ones in `openshift-release-release-4.6-periodics.yaml` and `openshift-release-release-4.5-periodics.yaml` as I understand those are the one to be exposed to the test grid and there's no automated generation support yet, but they complain with `error: invalid configuration: you must define at least one test or image build in ‘tests’ or ‘images’`